tests/Auth/bot-authentication/Dialogs/MainDialog.cs

```diff
@@ -53,6 +54,8 @@ public MainDialog(IConfiguration configuration, ILogger<MainDialog> logger)
 
         private async Task<DialogTurnResult> PromptStepAsync(WaterfallStepContext stepContext, CancellationToken cancellationToken)
         {
+            //return await stepContext.BeginDialogAsync(nameof(OAuthPrompt), null, cancellationToken);
```

Comment:
Should we remove this?

Reply:
This is just test code in the solution for ease of debugging. I'm changing this project continuously.